.gitignore

```diff
@@ -6,6 +6,9 @@ node_modules/
 *.tsbuildinfo
 jupytercad/labextension
 
+# Version file is handled by hatchling
+jupytercad/_version.py
```

Comment:
good catch!

Reply:
I just realized that it is necessary. Hatch creates that file when installing the extension; I removed it because it is now managed by hatch. We still need to keep it in the gitignore.

I'll fix it in #103